etl/pypasar/omop/observation_utils/config.py

```diff
@@ -0,0 +1,94 @@
+class ObservationMappingConfig():
+    observation_id_mapping = {
+        "pasar": ["id", "session_startdate"],
+        "omop": "observation_id"
+    }
+
+    person_id_mapping = {
+        "pasar": "anon_case_no",
+        "omop": "person_id"
+    }
+
+    observation_date_mapping = {
+        "pasar": "session_startdate",
+        "omop": "observation_date"
+    }
+    value_as_string_mapping = {
+        "pasar": [
+            ("preop.riskindex", ["h_o_ihd", "h_o_chf", "h_o_cva", "dm_on_insulin", "hypertension",
+                                 "history_of_osa", "loud_snoring", "daytime_tiredness", "apnoea", "cpap_use"]),
+            ("preop.others", ["forget_prescribed_medications", "continence",
+                              "no_of_prior_hospital_admissions", "inhalation_burns", "tbsa"]),
+            ("preop.char", ["smoking_history", "pregnancy_gender",
+                            "alcohol_consumption", "presence_of_malignancy"]),
+            ("postop.discharge", ["days_postop"]),
+            ("postop.info", ["satisfaction_at_analgesia_removal"]),
+            ("postop.icu", ["resuscitation_status"]),
+        ],
+        "omop": "value_as_string"
+    }
+
+    observation_type_concept_id_mapping = {
+        "pasar": "",
+        "omop": "observation_type_concept_id"
+    }
+
+    visit_occurrence_id_mapping = {
+        "pasar": "session_id",
```

Comment:
To update logic

Reply:
Can confirm what needs to be updated for this mapping?
From the [document](https://github.com/sidataplus/PASAR_ETL_Spec/blob/main/observation.md), it seems like visit_occurrence_id is just a simple mapping to session_id

Reply:
Mentioned [here](https://teams.microsoft.com/l/message/19:cf1d1c096fb14b7fbcef22f547c33211@thread.v2/1727136341596?context=%7B%22contextType%22%3A%22chat%22%7D)